jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/MavenServerConnector.java

```diff
@@ -55,6 +58,8 @@
     private String name;
     private int port;
     private long idleTimeout;
+    private HostPort serverAuthority;
+    private SocketAddress localAddress;
```

Comment:
Can't you just use these from the delegate connector?

Reply:
delegate connector?  Let me review that code again, I didn't notice that.

Reply:
Fixed